api/segments/models.py

```diff
@@ -195,6 +197,11 @@ def does_identity_match(
         elif self.operator == MODULO:
             if matching_trait.value_type in [INTEGER, FLOAT]:
                 return self._check_modulo_operator(matching_trait.trait_value)
+        elif self.operator == IN:
+            if matching_trait.value_type == INTEGER:
```

Comment:
We can simplify this by using the `trait_value` property on the `Trait` model. You can see how that works [here](https://github.com/Flagsmith/flagsmith/blob/main/api/environments/identities/traits/models.py#L57). 

This code would therefore become: 

```python
elif self.operator == IN:
    return str(matching_trait.trait_value) in self.value.split(",")
```